<div class="container">
  <% if @post_image.errors.any? %>
    <%= @post_image.errors.count %>件のエラーにより保存できませんでした
    <% @post_image.errors.full_messages.each do |message| %>
      <%= message %>
    <% end %>
  <% end %>
  <div class="row no-gutters">
    <div class="col-md-5 col-sm-10 mx-2 mx-sm-auto text-center shadow-lg rounded px-0 my-5">
      <h2 class="bg-light py-4 mb-0">投稿編集</h2>
      <div class="form-body p-4">

        <%= form_with model: @post_image, local: true do |f| %>
          <%= attachment_image_tag @post_image, :image, class: "w-100" %>
          <%= f.attachment_field :image, class: "w-100" %>
          <div class="input-group mb-3 mt-4">
            <div class="input-group-prepend">
              <span class="input-group-text" aria-hidden="true"><i class="fas fa-hand-sparkles"></i></span>
            </div>
            <%= f.text_field :title, class:"form-control", placeholder:"タイトル" %>
          </div>
          <%= f.text_area :caption, class:"form-control", rows:"3", placeholder:"コメントここに" %>
          <hr>
          <%= f.submit '保存', class:"btn btn-lg btn-block btn-success mt-4" %>
        <% end %>
      </div>
    </div>
  </div>
</div>
